meshstack-hub/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "meshstack-hub",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test",
+    "lint": "ng lint",
+    "lint:fix": "ng lint --fix",
+    "serve:ssr:meshstack-hub": "node dist/meshstack-hub/server/server.mjs"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/common": "^19.2.0",
+    "@angular/compiler": "^19.2.0",
+    "@angular/core": "^19.2.0",
+    "@angular/forms": "^19.2.0",
+    "@angular/platform-browser": "^19.2.0",
+    "@angular/platform-browser-dynamic": "^19.2.0",
+    "@angular/platform-server": "^19.2.0",
+    "@angular/router": "^19.2.0",
+    "@angular/ssr": "^19.2.3",
```

Comment:
d: Do we want server side rendering, or not? What do you think?

Reply:
could be worth to try, it's better for SEO seems, since it is a public website and people should come across it. We can try?

Reply:
Sure. We can try it out. But I want mention this that you are aware ;)